config.yaml

```diff
@@ -27,6 +27,7 @@ module:
         source: http://pkgs.fedoraproject.org/cgit/modules/base-runtime.git
```

Comment:
better to switch to `https://src.fedoraproject.org/.....` (not related to this PR)